projects/adi_vna_8ch/zcu102/system_constr.xdc

```diff
@@ -0,0 +1,159 @@
+###############################################################################
+
+## Copyright (C) 2023-2023 Analog Devices, Inc. All rights reserved.
+
+### SPDX short identifier: ADIBSD
+
+###############################################################################
+
+###############################################################################
+# AD9083
+###############################################################################
+
+# ref clock
+set_property  -dict {PACKAGE_PIN  G8}                       [get_ports ref_clk0_p]         ; ##  D04  FMC_HPC0 GBTCLK0 M2C_C_P
```

Comment:
The constraints need to be generate with the projects/scripts/adi_fmc_constr_generator.tcl script.
And I'm not sure if we should keep commented code in the file...